launchable/plugin.py

```diff
@@ -15,6 +16,8 @@ class Launchable(Plugin):
     # Grab stdout before the capture plugin
     score = Capture.score + 1
 
+    RAISE_ERROR_KEY = "LAUNCHABLE_RAISE_ERROR"
```

Comment:
The environment variable name is not fixed and we need to use the same name with the CLI. If you have a better idea for the name I'm happy to hear that.

Reply:
Hmm, I don't have any strong recommendations. My ideas are `LAUNCHABLE_ENABLE_ERROR` and `LAUNCHABLE_REPORT_ERROR`. If you like whichever, you would use it.